MastodonSDK/Sources/MastodonUI/Vendor/MetaTextView+PasteExtensions.swift

```diff
@@ -13,17 +13,21 @@ extension MetaTextView {
     public override func paste(_ sender: Any?) {
         super.paste(sender)
         
-        var nextResponder = self.next;
-        
-        // Force the event to bubble through ALL responders
-        // This is a workaround as somewhere down the chain the paste event gets eaten
-        while (nextResponder != nil) {
-            if let nextResponder = nextResponder {
-                if (nextResponder.responds(to: #selector(UIResponderStandardEditActions.paste(_:)))) {
-                    nextResponder.perform(#selector(UIResponderStandardEditActions.paste(_:)), with: sender)
+        // fix #660
+        // https://github.com/mastodon/mastodon-ios/issues/660
+        if #available(iOS 15.0, *) {
```

Comment:
Instead of enabling this one feature for iOS >15, we could also think about supporting iOS >15 for the entire app, so just the two most recent iOS-versions. What do you think? cc @kimar @Gargron

Reply:
That could be cool! iOS 15 also lets us use things like `AsyncPublisher` to do interop between async sequences and Combine.

Reply:
👍 on iOS >= 15 for me as well. I like to pursue n-1 for supported OSes, and given iOS 15 is supported by such a large percentage of device, I personally don't s a reason not to bump.

Reply:
Sounds good to me

Reply:
If we update the minimal version in the next build. The iOS 14.x still could install the app but only for v1.4.7. 

Maybe we fix this bug and push as v1.4.8. Then update the min OS in v1.5.0. So this issue not bother old OS users.